app/services/current-user.ts

```diff
@@ -71,6 +104,14 @@ export default class CurrentUserService extends Service {
             }
         });
     }
+
+    getWaffle(this: CurrentUserService, feature: string) {
+        if (this.get('waffleLoaded')) {
```

Comment:
Might be worth checking `setWaffle.isRunning`, and return `setWaffle.last.then(...)` if so. Don't want to restart the task if we don't need to.